youtube_dl/extractor/asiancrush.py

```diff
@@ -63,23 +93,47 @@ def _real_extract(self, url):
                 r'/p(?:artner_id)?/(\d+)', player, 'partner id',
                 default='513551')
 
-        return self.url_result(
-            'kaltura:%s:%s' % (partner_id, kaltura_id),
-            ie=KalturaIE.ie_key(), video_id=kaltura_id,
-            video_title=title)
+        description = self._html_search_regex(r'<div class="description">(.+?)</div>', webpage, 'description', fatal=False, flags=re.DOTALL)
```

Comment:
Move flags into regex. Carry long lines.

Reply:
I'm confused about "Move flags into regex", I can't find a difference between the way this is implemented in other extractors.

Reply:
What difference are you even talking about?

Reply:
You said "Move flags into regex", I don't understand what you're asking me to do.

Reply:
Do you aware what flags are at all?